cylc/flow/task_events_mgr.py

```diff
@@ -867,35 +958,44 @@ def _process_event_email(self, schd: 'Scheduler', ctx, id_keys) -> None:
                 subject = "[%d task events] %s" % (
                     len(id_keys), schd.workflow)
         cmd = ["mail", "-s", subject]
+
         # From: and To:
         cmd.append("-r")
         cmd.append(ctx.mail_from)
         cmd.append(ctx.mail_to)
+
         # STDIN for mail, tasks
         stdin_str = ""
         for id_key in sorted(id_keys):
-            (_, event), point, name, submit_num = id_key
-            stdin_str += "%s: %s/%s/%02d\n" % (event, point, name, submit_num)
+            stdin_str += f'job: {id_key.tokens.relative_id}\n'
+            stdin_str += f'event: {id_key.event}\n'
+            stdin_str += f'message: {id_key.message}\n\n'
+
         # STDIN for mail, event info + workflow detail
         stdin_str += "\n"
-        for key in (
-            WorkflowEventData.Workflow.value,
-            WorkflowEventData.Host.value,
-            WorkflowEventData.Port.value,
```

Comment:
Fix a small bug where the port was never provided to the event handler as it had changed from `Scheduler.port` to `Scheduler.server.port`.